recipes/pdfcombine/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pdfcombine" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "23e3af619a4859008745ac61a6040a230023cd4e5c4c667d9b8ea248726f47ed" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tdegeus/pdfcombine/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
```

Comment:
Why are you skipping `win`? Is `bld.bat` not working?

Reply:
@npavlovikj Thanks for your review! The problem were conflicting dependencies:
```none
The following specifications were found to be in conflict:
1771  - ghostscript
1772  - python[version='>=2.7,<2.8.0a0'] -> vc[version='>=9,<10.0a0']
```

Reply:
Does `pdfcombine` work with Python 2 and/or Python 3?

Reply:
I just tried 15 minutes ago, it only works with Python 3, so I now switched to 
```yaml
  skip: True  # [py<35]
```